tvm_linker/src/main.rs

```diff
@@ -98,7 +98,8 @@ fn linker_main() -> Result<(), String> {
             (@arg SETKEY: --setkey +takes_value conflicts_with[GENKEY] "Loads existing keypair from the file")
             (@arg WC: -w +takes_value "Workchain id used to print contract address, -1 by default.")
             (@arg DEBUG: --debug "Prints debug info: xref table and parsed assembler sources")
-            (@arg LIB: --lib +takes_value ... number_of_values(1) "Standard library source file. If not specified lib is loaded from environment variable TVM_LINKER_LIB_PATH if it exists.")
```

Comment:
Please, don't remove `... number_of_values(1)`. It breaks an important feature: multiple source files support. It works like `--lib lib1 --lib lib2 ...`.

Reply:
Rolled back. Thanks for the explanation.

Reply:
Thanks.